### DISCUSSION TITLE
Trying to install from unraid store - getting error - invalid mount config for type "bind": invalid mount path: 'WS_DATA_PATH' mount path must be absolute.

### DISCUSSION THREAD

**Discussion by vw-kombi** [arabcoders/watchstate#370] [General] [answered]
See below :

docker: Error response from daemon: invalid volume specification: '/config:WS_DATA_PATH:rw': invalid mount config for type "bind": invalid mount path: 'WS_DATA_PATH' mount path must be absolute.
See 'docker run --help'.

![image](https://github.com/ArabCoders/watchstate/assets/56822460/ce175783-8abe-4a58-abae-ea90e7f2e412)

**Comment by arabcoders** [comment, marked as answer]
Sorry about that, there was a mistake in the template. it should be fixed soon, wait until the next app store sync it will be synced, in the meantime you can replace the config target to `/config`

**Comment by vw-kombi** [comment]
No worries.  Hope to try it out tomorrow.  My fit may be different however as I want to be from primary emby to backup emby server, and the backup emby server will be as part of a script (wake up, copy latest media, then sync watch states). So a separate unraid server.  Not sure if that’s doable untill I see your questions on the setup for the backends.

**Reply by arabcoders** [reply]
Oh, it's no big deal, you can use the on demand syncing it's very doable
